addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -158,6 +158,12 @@ export class Powerbox {
         );
     }
 
+    // todo: remove in master. It has been kept when changing addEventListener
+    // keydown to input in stable to avoid removing a public method.
```

Comment:
While you're at it, I would rephrase to be more specific about the versions. I always find it strange to work in older versions and read things about "master". Which master is it?
So I'd say 15.0 instead of stable, and 15.3 instead of master (unless it's still time to merge in 15.2).

Reply:
Master is the right name as whatever the version gets frozen, it will always be in master that we could remove the code.

Reply:
I'm talking for posterity because this code will stay in 15.0 while master is not gonna remain the same. Anyway it doesn't matter enough to block this task, we can always eventually remove this comment.